File: birthday-cake-candles.js
function createAr(size){
    let arr = [];
    for(let i=0; i<size; i++){
        arr.push(parseInt(Math.random() * (111 - 1) + 1));
    }
    return arr;
}


function maxDuplicated(ar) {
    let max = Math.max(...ar);
    return ar.filter((v) => v === max).length;
}

console.log(maxDuplicated(createAr(99999)));

File: birthday-cake-candles.py
def bigCandles(ar):
	taller = max(ar)
	counter = 0
	for x in ar:
		if x == taller:
			counter += 1
	return counter

arr = [4,4,3,2,6,4,5]
res = bigCandles(arr)
print(res)